actions/ansible-docs-build-html/action.yml

```
@@ -58,7 +58,7 @@ runs:
         if [[ "$COPY_BUILD" != "" ]] ; then
             echo "::group::Copy the build files"
             mkdir -p "$COPY_BUILD"
-            rsync -avc --delete-after "$HTML" "$COPY_BUILD"
+            rsync -avc --delete-after "$HTML/" "$COPY_BUILD"
```

Why not also:
```suggestion
            rsync -avc --delete-after "$HTML/" "$COPY_BUILD/"
```
To be on the safe side? That one might also not have a trailing slash.

---

hm, sure why not